lib/godwoken_explorer/graphql/resolovers/transaction.ex

```diff
@@ -165,11 +165,24 @@ defmodule GodwokenExplorer.Graphql.Resolvers.Transaction do
 
       {from_account, to_account} ->
         if input[:combine_from_to] do
-          query
-          |> where(
-            [t],
-            t.to_account_id == ^to_account.id or t.from_account_id == ^from_account.id
-          )
+          base_query =
+            query
+            |> where(
+              [t],
+              t.to_account_id == ^to_account.id or t.from_account_id == ^from_account.id
+            )
+
+          if to_account.type == :eth_user do
```

Comment:
Why not limit query to `to_account.type == :polyjuice_creator`

Reply:
For native_transfer query, you need to pass from_account and to_account both as eth_user account

> Why not limit query to `to_account.type == :polyjuice_creator`

Reply:
Got it